### PR TITLE
Cleanup info components

### LINKED ISSUE #40: Clean up all info components
Right now we have three different info components:

- src/components/MaterialInfo.js
- src/components/ModelInfo.js
- src/components/HDRIInfo.js

Even though all of these are basically the same, this could easily be only one component that showed the buttons by the type of asset it is

### PR DESCRIPTION
**Pull request by njm222**
This resolves #40 & the r3f part of #41

### CODE REVIEW

**Comment by SaraVieira** [comment]
Hey

Found a small bug

When you are in the model it's missing a little margin to the top

Github is not letting meupload images :/

File: src/components/AssetInfo.js [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    createMatcapCodeDownload: s.createMatcapCodeDownload,
+    createPBRCodeDownload: s.createPBRCodeDownload,
+  }))
+
+  const { createHDRIDownload } = useHDRIStore((s) => ({
+    createHDRIDownload: s.createHDRIDownload,
+  }))
+
+  useEffect(() => {
+    if (tab === 'r3f') createBuffer(router.query.name)
```
Comment (SaraVieira) [inline src/components/AssetInfo.js:57]: You can also check if the asset type is model :)

File: src/components/AssetInfo.js [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+        )}
+        <Category category={asset.category} path={`${assetType.value}s`} />
+        <Views views={asset.views} />
+        <div className='my-4'>
+          <Tabs tabs={tabs} />
+
+          {tab !== assetType.value && (
+            <DownloadButton
+              onClick={() => {
+                let promise = copy(createModelDownload())
```
Comment (SaraVieira) [inline src/components/AssetInfo.js:130]: Can you move this obnClick to a function at the top?

File: src/components/canvas/HDRI.js [hunk 3]
```diff
@@ -15,19 +15,13 @@ export default function MaterialComponent(props) {
...
+          <sphereBufferGeometry args={[2, 128, 32]} />
+          <meshStandardMaterial metalness={1} roughness={0} />
+        </mesh>
+        <OrbitControls ref={ref} autoRotate={controls.autoRotate} />
       </Suspense>
     </Canvas>
   )
 }
-
-useGLTF.preload('/suzanne.gltf')
```
Comment (SaraVieira) [inline src/components/canvas/HDRI.js:33]: Thank you!

File: src/components/AssetInfo.js (hunk 1 above)
Comment (njm222) [reply src/components/AssetInfo.js:57]: so like this:
```suggestion
    if (assetType.value === 'model' && tab === 'r3f') createBuffer(router.query.name)
```

**Comment by SaraVieira** [comment]
Looks great! Thank you!!

**Pull request merged**